README.md

```diff
@@ -1 +1,185 @@
-# React-capstone
\ No newline at end of file
+<div align="center">
+  <h1>📺 Country-Metrics</h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📖 Table of Contents
+
+- [About the Project](#about-project)
+  - [🛠️ Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [🔑 Key Features](#key-features)
+  - [🚀 Demo](#live-demo)
+  - [💻 Getting Started](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Run Tests](#run-tests)
+    - [Build](#build)
+    - [Webpack Development Server](#webpack-development-server)
+  - [👤 Author](#author)
+  - [🔮 Future Features](#future-features)
+  - [🤝 Contributing](#contributing)
+  - [⭐️ Show Your Support](#show-your-support)
+  - [🙏 Acknowledgments](#acknowledgments)
+  - [📝 License](#license)
+
+# 📖 About the Project <a name="about-project"></a>
+
+**Country-Metrics** is a single-page application built with React and Redux. It allows users to explore anime categories, view numeric values based on selected categories, and access detailed information about specific anime titles.
+
+## 🛠️ Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>🖥️ Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+    <li><a href="https://reactrouter.com/">React Router</a></li>
+    <li><a href="https://axios-http.com/">Axios</a></li>
+    <li><a href="https://tailwindcss.com/">Tailwind CSS</a></li> 
+  </ul>
+</details>
+
+
+<!-- FEATURES -->
+
+### 🔑 Key Features <a name="key-features"></a>
+
+### Home Page
+
+- Upon loading, the webapp displays a list of anime categories, each with corresponding numeric values fetched from the API.
+- Users can click on a category item to navigate to the details page for that category.
+
+### Details Page
+
+- The details page shows comprehensive information about the selected anime category.
+- A "Back" button allows users to return to the home page.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Demo <a name="live-demo"></a>
+
+- [🔗 Live Demo Link](https://clarenceatim.github.io/Country-Metrics/)
+- [🔗 Presentation link](https://www.loom.com/share/f16a7b3d91ca4ea09a414822133287a2)
```

Comment:
- [ ] Kindly make sure that your  video presentation has audio, please make sure you follow all the [ presentation guidelines](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#project-documentation) when recording your next video  👍🏿